app/templates/macros/nav_macros.html

```diff
@@ -50,11 +73,23 @@
  # secondary menu. `count` should be the string (e.g. 'four') number of endpoints. #}
 {% macro desktop_nav(current_user, endpoints=None, count=None) %}
     <div class="computer tablet only row">
-        <div class="ui fixed inverted black main menu">
+        <div class="ui fixed inverted red main menu">
             <div class="ui container">
                 {{ header_items(current_user) }}
+
```

Comment:
do we want this blank line here? I think it's fine if you're looking for some separation, but then we should probably be consistent with the other times we use jinja followed by html.